### DISCUSSION TITLE
Pterodactyl Email change

### DISCUSSION THREAD

**Discussion by PotionsYT** [pterodactyl/panel#3951] [Support]
Is it posible to change the contents of the email. Because when i send the email to somone they get localhost on the url. Is there a way to remove or edit the contents of the email?

**Comment by trenutoo** [comment]
The URL is determined by your application URL, which you'd configure during setup. Changing afterward can be done by opening the environment file by default located at `/var/www/pterodactyl.env`. 

Restart the queue worker once the APP_URL is modified to apply it `systemctl restart pteroq`

---

As to editing the email templates, all templates are located at `/resources/views/vendor/notifications` and `/app/Notifications`

**Reply by PotionsYT** [reply]
These are the files that show up in that locaion
![image](https://user-images.githubusercontent.com/62046235/153312152-12ac351e-04dc-4156-9c41-3fb7ce63cc63.png)

**Reply by PotionsYT** [reply]
Both do not have what im looking for.

**Reply by trenutoo** [reply]
The two locations I specified contain everything related to the sent mail content.

**Reply by PotionsYT** [reply]
Well it seems to be right. But i cant change the URL. 
It just says actionURL

**Reply by trenutoo** [reply]
Please read my original reply regarding app url. You don't have to modify the template for it.

**Reply by PotionsYT** [reply]
Nothing is motified though. so why would i need to restart it.

**Reply by PotionsYT** [reply]
This is what happend.

**Reply by trenutoo** [reply]
Your question was regarding how to change the URL. Please read my original reply, which explains it.

**Reply by PotionsYT** [reply]
I am not understanding. I am very new to this. It takes me 5 hours to get pteractyl. Up

**Reply by PotionsYT** [reply]
![image](https://user-images.githubusercontent.com/62046235/153412507-de1cb5ef-c456-4030-9163-fedf01e08cd2.png)

**Reply by PotionsYT** [reply]
Its set using the IPV4 my public ip.

but no local host. 
I dont know where this local host is coming from

**Comment by PotionsYT** [comment]
![image](https://user-images.githubusercontent.com/62046235/153411071-f4d47a5a-ffd2-47b3-9d94-acba35a31079.png)
